lib/capybara/cuprite/javascripts/index.js

```diff
@@ -115,6 +115,18 @@ class Cuprite {
     return `//${selectors.join("/")}`;
   }
 
+  isObscured(node, x, y) {
+    let rect = node.getBoundingClientRect();
+    if (x == null) x = rect.width/2;
+    if (y == null) y = rect.height/2;
+
+    let px = rect.left + x;
+    let py = rect.top + y;
+    let e = document.elementFromPoint(px, py);
+
+    return !node.contains(e) || { x: px, y: py };
```

Comment:
I'm not following the `{ x: px, y: py }` return type. `isObscured` should be a boolean type, by the naming, but here we see the object return. It's also not clear for me what this object means.

Reply:
`isObscured` now always returns a boolean.